yellowbrick/text/tsne.py

```diff
@@ -117,12 +134,15 @@ def tsne(
     # Fit and transform the visualizer (calls draw)
     visualizer.fit(X, y, **kwargs)
     visualizer.transform(X)
-    visualizer.finalize()
+
```

Comment:
Can you change the parameter `classes` in the quick method to `labels`.  This will ensure the docstring of the quick method matches that of the visualizer.